src/core/container.jl

```diff
@@ -245,21 +243,3 @@ function resample!(
 
     pc
 end
-
-
-########### Auxilary Functions ###################
-
-# ParticleContainer: particles ==> (weight, results)
-function getsample(pc :: ParticleContainer, i :: Int, w :: Float64 = 0.)
```

Comment:
I notice this function is removed in this PR. Any particular reason?

Reply:
When I started cleaning the implementation of `getsample` I noticed that actually it's never called. So I just removed it.

Reply:
Can you just comment them out for now instead of deleting? Unless @trappmartin or @yebai can confirm this is not useful at all.

Reply:
I can confirm this is no longer needed after the MCMC interface work #793